pyvlx/api/frames/frame_get_limitation.py

```diff
@@ -70,29 +78,34 @@ class FrameGetLimitationStatusNotification(FrameBase):
 
     PAYLOAD_LEN = 10
 
-    def __init__(self):
+    def __init__(self) -> None:
         """Init Frame."""
         super().__init__(Command.GW_LIMITATION_STATUS_NTF)
-        self.session_id = None
+        self.session_id: Optional[int] = None
         self.node_id = 0
         self.parameter_id = 0
-        self.min_value = None
-        self.max_value = None
-        self.limit_originator = None
-        self.limit_time = None
+        self.min_value: Optional[bytes] = None
+        self.max_value: Optional[bytes] = None
+        self.limit_originator: Optional[Originator] = None
+        self.limit_time: Optional[int] = None
 
-    def get_payload(self):
+    def get_payload(self) -> bytes:
         """Return Payload."""
+        assert self.session_id is not None
+        assert self.min_value is not None
+        assert self.max_value is not None
+        assert self.limit_originator is not None
+        assert self.limit_time is not None
         payload = bytes([self.session_id >> 8 & 255, self.session_id & 255])
         payload += bytes([self.node_id])
         payload += bytes([self.parameter_id])
-        payload += bytes([self.min_value >> 8 & 255, self.min_value & 255])
-        payload += bytes([self.max_value >> 8 & 255, self.max_value & 255])
-        payload += bytes([self.limit_originator])
+        payload += self.min_value
+        payload += self.max_value
+        payload += bytes([self.limit_originator.value])
```

Comment:
This is an actual logic case. I'm not totally sure what the right way around is here, but either `min/max_value` are bytes or they're ints, and I think bytes makes most sense given the rest of the code. Similarly for limit_originator being an int or an actual `Originator`.